package.json

```diff
@@ -38,14 +38,14 @@
     "@hot-loader/react-dom": "^16.8.6",
     "@khanacademy/flow-to-ts": "^0.1.9",
     "@microsoft/api-extractor": "^7.13.0",
-    "@storybook/addon-a11y": "@next",
-    "@storybook/addon-actions": "@next",
-    "@storybook/addon-docs": "@next",
-    "@storybook/addon-essentials": "@next",
-    "@storybook/addon-links": "@next",
-    "@storybook/addons": "@next",
-    "@storybook/react": "@next",
-    "@storybook/theming": "@next",
+    "@storybook/addon-a11y": "6",
```

Comment:
@clxandstuff  hmm I am wondering why yarn specified it that way instead of `^6.3.8`. Do you know the reason?

Reply:
I think this is my choice :) I will install them without specifying version. This should update package.json  to proper version

Reply:
Done :)